src/utils/oauth2_util.py

```diff
@@ -473,3 +479,30 @@ def build_authentication_response(secret_key, access_token, refresh_token):
         authentication_response.update({"user": decoded_token.get("user")})
 
     return authentication_response
+
+
+def build_refresh_authentication_response(secret_key, access_token):
```

Comment:
This is doing something pretty similar to: `build_authentication_response`, could you optimize it?

Reply:
done.